dijet/tasks/correlated_fit.py

```diff
@@ -0,0 +1,89 @@
+# coding: utf-8
+
+import law
+from columnflow.util import maybe_import
+
+from dijet.tasks.util import chi2
+
+sc = maybe_import("scipy.optimize")
+nd = maybe_import("numdifftools")
+np = maybe_import("numpy")
+
+class CorrelatedFit():
+    """
+    Task to calculated correlated fit to widths.
+
+    ## Covariance matrix 
+    [1] Based on K. Goebel dissertation ch. A.3.
+    https://www.physik.uni-hamburg.de/en/iexp/gruppe-haller/scientific-output/documents/thesis-kristin-goebel.pdf
+
+    Example code here
+    https://github.com/UHH2/DiJetJERC/blob/ff98eebbd44931beb016c36327ab174fdf11a83f/
+    JERSF_Analysis/JER/wide_eta_binning/functions.C#L558
+    func. createCov()
+    """
+
+    def createCov(self, widths, widths_err):
+        widths_err2 = pow(widths_err, 2)  # square each element
```

Comment:
Let's use the Python notation for exponentiation here.
```suggestion
        widths_err2 = widths_err**2  # square each element
```